docs/about.md

```diff
@@ -1,22 +1,14 @@
-# About Heptio Ark
+# How Ark Works
 
-Heptio Ark provides customizable degrees of recovery for all Kubernetes objects (Pods, Deployments, Jobs, Custom Resource Definitions, etc.), as well as for persistent volumes. This recovery can be cluster-wide, or fine-tuned according to object type, namespace, or labels.
+Each Ark operation -- on-demand backup, scheduled back, restore -- is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. The config custom resource specifies core information and customized options such as cloud provider settings. These resources are handled by [custom controllers][21] on the Kubernetes API server.
```

Comment:
typo: scheduled back -> backup

Reply:
I know this is was there before, but would "customizable options" make more sense than "customized options"?

Reply:
"These resources are handled by custom controllers on the Kubernetes API server" doesn't sound right to me. What if we removed this sentence, and started the following paragraph with "Ark contains controllers that process its custom resources to execute backups, restores, etc. Each controller watches ..."? Or something like this?

Reply:
>1 comment on this one, so went with fewer fewer words. Happy to revisit if cut to excess.